doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -357,14 +357,19 @@ The device selector always has the form of ``DDDD:BB:DD.FF`` or ``DDDD.BB.DD.FF`
 
 and then set::
 
-  bluestore_block_path = spdk:0000:01:00.0
+  bluestore_block_path = "spdk:trtype:PCIe traddr:0000:01:00.0"
```

Comment:
probably deserves a line in release notes despite that the SPDK backend is not an officially support feature.

Reply:
ack